static/js/lib/site_content.ts

```diff
@@ -97,7 +97,8 @@ export function widgetExtraProps(field: ConfigField): Record<string, any> {
   case WidgetVariant.Markdown:
     return {
       minimal: field.minimal ?? false,
-      attach:  field.attach ?? undefined
+      link:    field.link ?? [],
+      embed:   field.embed ?? []
```

Comment:
if we're always going to return `[]` here then I think we could probably type `link` and `embed` on the Markdown field props like `link: string[]` rather than like `link?: string[]`